WG.md

```diff
@@ -29,6 +29,18 @@ Members: Colm Ryan, Andrew Cross, Ali Javadi, Blake Johnson, Matthew Amy
 Chair: Michael Healy (IBM Quantum)  
 Members: Niel de Beaudrap, Hiroshi Horii, Blake Johnson, Colm Ryan, Luciano Bello, Prasahnt Sivarajah, Yunong Shi, Jake Lishman
 
+### Pragmas
+
+**Objective**: Define a pragma syntax and standard directives   
+**Questions**:
+
+ * What is the pragma syntax to define custom directives
+ * what is not supported by pragmas and should be supported by the language itself
+ * Which use cases from https://github.com/Qiskit/openqasm/issues/294 are covered?
+
+Chair: Jeff Heckey (Amazon)  
+Members: 
```

Comment:
```suggestion
Chair: Jeff Heckey (AWS)  
Members: 
```suggestion
Members: Jeff Heckey (AWS), Ali Javadi (IBM Quantum)
BẢO Bach Gia (Ho Chi Minh University of Technology), Blake Johnson (IBM Quantum), Jake Lishman (IBM Quantum), Jack Woehr (IBM Quantum), Lev S. Bishop (IBM Quantum), Michael Healy (IBM Quantum), Thomas Alexander (IBM Quantum), Yunong Shi (AWS)

```

```